apps/laboratory/tests/shared/validators/ModalValidator.ts

```diff
@@ -83,7 +83,7 @@ export class ModalValidator {
   }
 
   async expectCaipAddressHaveCorrectNetworkId(caipNetworkId: CaipNetworkId) {
-    const address = this.page.getByTestId('appkit-caip-address')
+    const address = this.page.getByTestId('w3m-caip-address')
```

Comment:
Also here

Reply:
This is just aligning the test ids on laboratory with the other test id's - these are not the ids in AppKit itself. 

I'd like to change all but that wouldn't make sense to replace all `w3m` calls in this PR I guess